dbms/src/Functions/tests/gtest_strings_position.cpp

```diff
@@ -42,136 +42,111 @@ class StringPosition : public DB::tests::FunctionTest
 TEST_F(StringPosition, strAndStrTest)
 {
     const auto context = TiFlashTestEnv::getContext();
```

Comment:
Seems changes to this file is not necessary for current pr?

Reply:
> Seems changes to this file is not necessary for current pr?

I will put it into another pr